src/test/java/com/virtuslab/archunit/ForbiddenClassesTestSuite.java

```diff
@@ -10,6 +10,7 @@ public class ForbiddenClassesTestSuite extends BaseArchUnitTestSuite {
   public void no_classes_should_depend_on_java_collections() {
     noClasses()
         .that().resideOutsideOfPackages(
+            "com.virtuslab.gitmachete.backend.hooks",
             "com.virtuslab.gitmachete.frontend.actions..",
```

Comment:
why are there two subsequent dots `..` sometimes? 🤔

Reply:
`foo.bar..` is `foo.bar` and all its subpackages 📦